docs/applications/cli.md

```diff
@@ -0,0 +1,56 @@
+---
+id: cli
+title: bitcoin-s cli
+---
+
+
+## Bitcoin-s command line client
+
+The [cli](../../app/cli/) project is meant to be a bitcoin-s command line client. It uses [graalvm native image](https://www.graalvm.org/docs/reference-manual/native-image/) to
+create a native executable that circumvents the jvm for fast start up time.
+
+### Building the command line
+
+#### Installing graalvm native image
+First to build the command line you need to have the graal jdk installed. This can be installed on [graalvm's github](https://github.com/graalvm/graalvm-ce-builds/releases/tag/vm-19.3.0).
+
+
+If you do not have the `native-image` executable installed, you need to install it with
+
+```bashrc
+./graalvm-ce-java8-19.3.0/bin/gu install native-image
+```
+
+Once you have the graalvm installed you should be able to verify you have the `native-image` exeuctable installed by running
+
+```bashrc
+$ native-image --help
+```
+
+#### Building the native image
+
+Now that you have graalvm native image installed, you should be able to build the cli with. It is important to note that
+this only works with Scala `2.12.x`. I'm unsure of why the native image build fails with Scala `2.13.x`
+
+```bashrc
+$ sbt ++2.12.10 cli/graalvm-native-image:packageBin
+```
+
+After running that command you should find the executable here:
+
+```bash
+bitcoin-s/app/cli/target/graalvm-native-image
+```
+
+#### Executing commands
+You can ask the client for help with
```

Comment:
does this command also provide a list of available commands? If so we should mention that it does that too

Reply:
It does not, even though it claims it does.

Reply:
https://github.com/bitcoin-s/bitcoin-s/issues/912